src/main/java/FridayExceptions/EmptyListException.java

```diff
@@ -0,0 +1,4 @@
+package FridayExceptions;
```

Comment:
Maybe have this in lower case? Following Java coding standards: https://se-education.org/guides/conventions/java/basic.html#naming

Similarly for the rest of the `Exception` classes.